js/main.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+var ADVERT_NUMBERS = ['1', '2', '3', '4', '5', '6', '7', '8'];
+var ADVERT_PRICE = ['300', '400', '500', '600', '700', '800', '900', '1000'];
+var ADVERT_TYPE = ['palace', 'flat', 'house', 'bungalo'];
+var ADVERT_CHECKIN_CHECKOUT = ['12:00', '13:00', '14:00'];
+var ADVERT_FEATURES = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
+var ADVERT_PHOTOS = ['http://o0.github.io/assets/images/tokyo/hotel1.jpg', 'http://o0.github.io/assets/images/tokyo/hotel2.jpg', 'http://o0.github.io/assets/images/tokyo/hotel3.jpg'];
+
+var getRandomNumberFromRange = function (min, max) {
+  return Math.round(Math.random() * (max - min) + min);
+};
+
+var getRandomItemArray = function (arr) {
+  var index = getRandomNumberFromRange(0, arr.length - 1);
+  var result = arr[index];
+  arr.splice(index, 1);
+
+  return result;
+};
+
+var getRandomIndexArray = function (arr) {
+  var min = 0;
+  var max = arr.length - 1;
+  var result = Math.round(Math.random() * (max - min) + min);
+
+  return arr[result];
+};
+
+var getArrayRandomLength = function (arr) {
+  var newArray = [];
+  var number = getRandomNumberFromRange(1, arr.length - 1);
```

Comment:
Может тут должно быть так ?
```suggestion
  var number = getRandomNumberFromRange(0, arr.length - 1);
```